### ISSUE TITLE
The < and > are too small, should be closer to the height of [ and ]

### ISSUE THREAD

**Issue by Multiforge** [madmalik/mononoki#41]
Hi, this looks like a great font. I've downloaded it and like it very much. Thanks for making it available for free.

However, I do have one small comment. Can you make the < and > glyphs taller? In a lot of programming languages the < and > are used to delimit code, such as in Java,

    List<String> list = new ArrayList<String>();

or Apple's new Swift language,

    func returnZero() -> Int { return 0 }

or in XHTML,

    <div id="..."> ... </div>

TLDR; having "short" < and > makes it harder to see them during programming. The < and > aren't just for number comparison, they're part of the languages. IMHO they should be roughly 80% of the height of [ and ] in order to properly delimit code. Please consider making this change for the benefit of programmers.

**Comment by madmalik** [comment]
Hi

This is an interesting idea. I've made a mockup of the visual difference:

<img width="209" alt="default" src="https://cloud.githubusercontent.com/assets/1328396/15747076/6e562a04-28d9-11e6-91cd-f33aeaf38d69.png">

I'm not sure about it. I don't like the bigger less and bigger signs (:P) not so much in arithmetic expressions, but I think you are right that they look better as delimiters. And while thinking about it, there are some precedents for such designs, especially in pixel fonts for terminals like Fixedsys or Dina.

Maybe some other people can tune in.

**Comment by jdw1996** [comment]
I'm trying out this modified version right now. I like the bigger ones better as delimiters, in bit shifts, and for printing in C++, but I don't like them at all as less than/greater than signs. Perhaps there is some middle ground that would look good in both contexts?

**Comment by Multiforge** [comment]
>  Perhaps there is some middle ground that would look good in both contexts?

Good point. Ya, the second version does look a bit too large. 

Maybe halfway between the original implementation and the new version would be the optimal height for programming?

**Comment by madmalik** [comment]
Yeah, I agree, they are to big (thats intentional. I like to exaggerate new ideas in the first iteration and tone them down after that). 
Another problem is, that the other delimiters are centered a little bit higher than operators. 

One possibility would be this:
<img width="335" alt="default" src="https://cloud.githubusercontent.com/assets/1328396/15773459/3d44e01a-2977-11e6-9a4d-3961b5e59db6.png">

The "openness" is in between the two versions and it's set a little bit higher, so that it aligns better with the other delimiters. That results in arrows that are slightly off center... not sure about that.

**Comment by jdw1996** [comment]
I quite like those ones. I think they're the best of the three.

As for lining up with other delimiters, I think it's more important that they line up with the operators, for arrows as well as constructs like "greater than or equal to" (i.e. `>=`). That being said, I use `<` and `>` as operators more than delimiters, so I am biased toward that line of thinking.

**Comment by Multiforge** [comment]
> I quite like those ones. I think they're the best of the three.

Same here.

**Comment by madmalik** [comment]
I've used the "comprise" some time now and still like it. The only change is the centering — the original operator-height just works best. They are in the latest release (alongside box drawing characters, wohoo!)

Big thanks to @Multiforge for the suggestion and @jdw1996 for the help!
